shelterAPI.json

```diff
@@ -103,6 +100,38 @@
           }
         }
       }
+    },
+    "/shelter/announcements": {
+      "get": {
+        "tags": [
+          "Endpoints"
+        ],
+        "summary": "Gets all announcements for shelter. Requires shelter role. Gets shelter id from auth claims.",
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "404": {
+            "description": "Client error"
+          }
+        }
+      }
+    },
+    "/shelter/pets": {
+      "get": {
+        "tags": [
+          "Endpoints"
+        ],
+        "summary": "Gets all pets for shelter. Requires shelter role. Gets shelter id from auth claims.",
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "401": {
+            "description": "Client error"
```

Comment:
This probably should say something like `Unauthorized`. Also (I think) by default ASP.Net Core returns `403` if a user does not have a permission to some resource (since for this user the resource is permanently unavailable), and `401` if a user is unauthenticated, so we could add this status code here.

Reply:
Same as before, Unauthorized is much better choice.